docs/user-interfaces/elements/drop-target.mdx

```diff
@@ -1 +1,104 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+import UppyCdnExample from '/src/components/UppyCdnExample';
+
 # Drop target
+
+The `@uppy/drop-target` plugin lets your users drag-and-drop files on any
+element on the page, for example the whole page, `document.body`.
+
+Can be used together with Uppy Dashboard or Drag & Drop plugins, or your custom
+solution, including plain text “please drop files here”.
+
+:::tip
+
+[Try out the live example](/examples) or take it for a spin in
+[CodeSandbox](https://codesandbox.io/s/uppy-drag-drop-gyewzx?file=/src/index.js).
```

Comment:
/examples doesn't exit. We should also create a codesandbox for this plugin and link it here.

Reply:
We plan to have something at `/examples` don't we?

https://github.com/transloadit/uppy.io/blob/0f9e149b61682ae41579712e346cc98698970556/docusaurus.config.js#L58

Reply:
No as discussed we will go live without it and potentially never have it as we emphasize our existing examples repo and codesandboxes. So at least for now we need to remove it.

Reply:
> We should also create a codesandbox for this plugin and link it here.

I don't know what's needed for this. We should probably document some guidelines around it.

> No as discussed we will go live without it and potentially never have it as we emphasize our existing examples repo and codesandboxes.

There are 12 other pages linking to `/examples`, plus the main nav menu cited above. I don't think it makes sense to remove it from this PR. If we plan to remove them before going live, it can be done in a follow-up PR and doesn't have to block this one.